tests/src/core/testqgsvectorlayer.cpp

```diff
@@ -334,7 +334,7 @@ void TestQgsVectorLayer::testAddTopologicalPoints()
   QVERIFY( layerLine->isValid() );
 
   QgsPolylineXY line1;
-  line1 << QgsPointXY( 2, 1 ) << QgsPointXY( 1, 1 ) << QgsPointXY( 1, 3 );
+  line1 << QgsPointXY( 2, 1 ) << QgsPointXY( 1, 1 ) << QgsPointXY( 1, 5 );
```

Comment:
What's the reason to change 3 by 5?

Reply:
Just added 2 units to the Y coordinate so that it gave me more space to attempt to add topological points from a QgsGeometry, still using integer values.

Reply:
OK. Thanks